integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/util/ShowClusterDefinitionUtil.java

```diff
@@ -0,0 +1,47 @@
+package com.sequenceiq.it.cloudbreak.newway.util;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Strings;
+import com.sequenceiq.it.cloudbreak.exception.TestFailException;
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+import com.sequenceiq.it.cloudbreak.newway.entity.stack.StackTestDto;
+
+public class ShowClusterDefinitionUtil {
+
+    private ShowClusterDefinitionUtil() {
+
+    }
+
+    public static StackTestDto checkFutureClusterDefinition(TestContext testContext, StackTestDto stackTestDto, CloudbreakClient cloudbreakClient) {
+        String extendedClusterDefinitionText = stackTestDto.getGeneratedClusterDefinition().getClusterDefinitionText();
+        validateGeneratedClusterDefinition(extendedClusterDefinitionText);
+        return stackTestDto;
+    }
+
+    public static StackTestDto checkGeneratedClusterDefinition(TestContext testContext, StackTestDto stackTestDto, CloudbreakClient cloudbreakClient) {
+        String extendedClusterDefinitionText = stackTestDto.getResponse().getCluster().getAmbari().getExtendedClusterDefinitionText();
+        validateGeneratedClusterDefinition(extendedClusterDefinitionText);
+        return stackTestDto;
+    }
+
+    private static void validateGeneratedClusterDefinition(String extendedClusterDefinitionText) {
```

Comment:
I would create a stronger assert here to test at least 1 field of the Json that is generated.